proto/dydxprotocol/sending/transfer.proto

```diff
@@ -48,6 +48,8 @@ message MsgDepositToSubaccount {
 // MsgWithdrawFromSubaccount represents a single transfer from an
 // `x/subaccounts` subaccount to an `x/bank` account.
 message MsgWithdrawFromSubaccount {
+  option (cosmos.msg.v1.signer) = "sender";
```

Comment:
I understand that we are specifying the sender should be `sender.owner` in `CustomGetSigner` [here](https://github.com/dydxprotocol/v4-chain/pull/867/files#r1431040322). 

Should we remove this option, given that `sender` is not a valid signer (i.e. it's not an address string)?

Reply:
The cosmos SDK was meant to support these recursively but didn't do it quite right since `sender` is supposed to go into that message and then use `owner` to find the appropriate address string. I filed https://github.com/cosmos/cosmos-sdk/issues/18722 and they have fixed this but it won't be available till a future Cosmos SDK release.

I'd like to leave this for now and once able we can remove the `CustomGetSigners` implementation.